tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1557,15 +1556,20 @@ schedule(void)
       merge_next_block = false;
     }
 
-    if (XBIT(183, 0x10000000)) {
-      if ((!XBIT(69, 0x100000)) && BIH_NODEPCHK(bih) &&
-          (!ignore_simd_block(bih))) {
-        fix_nodepchk_flag(bih);
-        mark_rw_nodepchk(bih);
-      } else {
-        clear_rw_nodepchk();
-      }
+    open_pragma(BIH_LINENO(bih));
+    BIH_NODEPCHK(bih) = !flg.depchk;
+    if(XBIT(19, 0x18))
+      BIH_NOSIMD(bih) = true;
+    else if(XBIT(19, 0x400))
```

Comment:
Please insert a space between `if` and the left parenthesis, on both lines 1561 and 1563.

Reply:
Done.